docs/shared/reference/nx-json.md

```diff
@@ -507,6 +510,32 @@ The `git` property configures the automated git operations that take place as pa
 }
 ```
 
+## Sync
+
+These are global configuration options for the [`nx sync`](/reference/nx-commands#sync) command. The `nx sync` command runs all global and task-specific [sync generators](/concepts/sync-generators) to keep applicable files in sync with your project graph.
```

Comment:
Same here about the sync generators not being limited to using the project graph as the source of truth.